paymentsheet-example/src/androidTestDebug/java/com/stripe/android/TestHardCodedLpms.kt

```diff
@@ -265,4 +265,44 @@ class TestHardCodedLpms {
             }
         )
     }
+
+    @Test
+    fun testCashAppPay_Success() {
+        testDriver.confirmNewOrGuestComplete(
+            testParameters = newUser.copy(
+                paymentMethod = lpmRepository.fromCode("cashapp")!!,
+                currency = Currency.USD,
+                merchantCountryCode = "US",
+                authorizationAction = AuthorizeAction.Authorize,
+                supportedPaymentMethods = listOf("card", "cashapp"),
+            ),
+        )
+    }
+
+    @Ignore("The test page currently still returns a success code when tapping 'Fail'. Wait until this is fixed.")
+    @Test
```

Comment:
I’ll enable them once the Stripe-hosted test pages work as expected again. Currently, they still return a success even if you press `Fail`.

Reply:
The Stripe-hosted test page was fixed, so these tests can run now.